packages/appsync-emulator-serverless/lambdaRunner.js

```diff
@@ -1,39 +1,39 @@
 const log = require('logdown')('appsync-emulator:lambdaRunner');
 
+const sendOutput = output => {
+  process.send({ type: 'success', output }, process.exit);
+};
 const sendErr = err => {
-  process.send({ type: 'error', error: err }, () => {
-    process.exit();
-  });
+  process.send({ type: 'error', error: err.message ? err.message : err }, process.exit);
```

Comment:
I think we want to always send `err` here so that we can throw an error [here](https://github.com/ConduitVC/aws-utils/blob/8bff616801546916282664086b24b3ce3070b3d4/packages/appsync-emulator-serverless/lambdaSource.js#L59)

Reply:
It looks impossible to send `Error` instance to another process.

In real AWS lambda, it returns a below json string when `throw new Error('test error')` or `callback(new Error('test error'), null)` is.

```
{
  "errorMessage": "test error",
  "errorType": "Error",
  "stackTrace": [
    "exports.handler (/var/task/index.js:8:11)"
  ]
}
```

Reply:
It is not impossible to send an `Error` instance to another process.

```
Note: The message goes through serialization and parsing. The resulting message might not be the same as what is originally sent.
```

See [here](https://nodejs.org/docs/latest-v8.x/api/child_process.html#child_process_subprocess_send_message_sendhandle_options_callback).  For more information.

Regarding real AWS lambda, that should be mimicked in `lamdaSource`, not in the Runner.  Feel free to make another PR which updates `lambdaSource` to do exactly what AWS lambda does in case of an error.

Reply:
If I send not `err.message` but `err`, `response.error` is `{}` in the test. That is same as `JSON.stringify(new Error('error'))`.

I think that `sendErr` manipulates an error for serilizing the error properties.

Reply:
My apologies for being unclear earlier.  I agree that `sendErr` should manipulate an error.

I'm thinking we first try to send `err.stack`, then `err.message`, then `err`